APMrover2/Rover.h

```diff
@@ -107,6 +108,7 @@ class Rover : public AP_HAL::HAL::Callbacks {
     friend class ParametersG2;
     friend class AP_Rally_Rover;
     friend class AP_Arming_Rover;
+    friend class AP_MotorsUGV;
```

Comment:
we don't need this friend anymore do we?

Reply:
done